srcpkgs/Carla/template

```diff
@@ -0,0 +1,25 @@
+# Template file for 'Carla'
+pkgname=Carla
+version=1.9.8
+revision=1
+build_style=gnu-makefile
+hostmakedepends="pkg-config"
+makedepends="
+ python3-PyQt5-devel-tools python3-PyQt5 libmagic file-devel liblo-devel
+ alsa-lib-devel pulseaudio-devel libX11-devel gtk+3-devel gtk+-devel qt5-devel
+ qt-devel fluidsynth-devel fftw-devel mxml-devel zlib-devel python3-rdflib"
```

Comment:
Any particular reason to enable support for qt4? Imo it should be disabled except if it is really needed.

Reply:
qt4 support is needed for full support of lv2 plugins. I'm not sure how many plugins actually use qt4 however.

Reply:
Ok. If it actually add functionality that you won't get with qt5 only then let it there. I just would prefer not adding to much new stuff that depends on qt4 since qt4 is EOL.